src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -65,20 +65,20 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
         [ColumnKey.FREQ]: 0,
     };
 
-    private reactions:IReactionDisposer[] = [];
+    private reactions: IReactionDisposer[] = [];
```

Comment:
looks like maybe you have a code formatter that adds spaces?

Reply:
Yes, I typically do use a code formatter, do we have existing linting/formatting settings that are project wise? I can use those instead if we have them.